CHANGELOG.md

```diff
@@ -4,9 +4,12 @@
 
 ## DFX
 
-
 ### fix: `cargo run -p dfx -- --version` prints correct version
 
+### chore: Kill unwanted CI jobs
+
+If pushing twice to the same branch in quick succession, the earlier CI jobs are cancelled in favour of the later ones.
+
```

Comment:
We track changes to dfx in the changelog, but not things like changes to our CI process.
```suggestion
```

Reply:
Ack.  Makes sense.  Undone.